lib/api_auth/base.rb

```diff
@@ -67,6 +67,8 @@ def generate_secret_key
 
   private
 
+    AUTH_HEADER_PATTERN = /^APIAuth ([^:]+):(.+)$/
```

Comment:
My only concern is the addition of the beginning of line match which didn't exist before. All the tests past, but this is a potentially breaking change I think. This might be fine though, why would there ever be whitespace or something else before the APIAuth, or is it a "feature" to allow for "SpecificAPIAuth"?